test/common-tests/request-field-validation-api-tests/non-negative-number-field-validation-api-tests.ts

```diff
@@ -73,6 +61,20 @@ export function withNonNegativeNumberFieldValidationApiTests<RequestBodyItem>({
       });
     }
 
+    it(`returns a 400 response if ${fieldName} is null`, async () => {
+      const requestWithNullField = { ...validRequestBody[0], [fieldNameSymbol]: null };
+      const preparedRequestWithNullField = prepareModifiedRequest(requestIsAnArray, requestWithNullField);
+
+      const { status, body } = await makeRequest(preparedRequestWithNullField);
+
+      expect(status).toBe(400);
+      expect(body).toMatchObject({
+        error: 'Bad Request',
+        message: expect.arrayContaining([expectedNonEmptyFieldError]),
+        statusCode: 400,
+      });
+    });
+
```

Comment:
Should we not also add a test for the other case (i.e., 2xx response if fieldName is null when required is false?)

Reply:
@oscar-richardson-softwire, I originally was going to do this but realised that we always prevent `null` for numbers (regardless of if it's required or not) and there's already a test for this in the file